docs/welcome.md

```diff
@@ -51,7 +48,7 @@ Additionally, you can browse the [.NET API reference](/dotnet/api).
 
```

Comment:
The link to the .API reference on line 47 is broken.

Reply:
I haven't changed that link and it works for me.